core/dbt/logger.py

```diff
@@ -20,20 +20,11 @@
 # then we should override the logging stream to use the colorama
 # converter. If the TERM var is set (as with Git Bash), then it's safe
 # to send escape characters and no log handler injection is needed.
-colorama_stdout = sys.stdout
-colorama_wrap = True
-
-colorama.init(wrap=colorama_wrap)
-
-
-if sys.platform == "win32" and not os.getenv("TERM"):
-    colorama_wrap = False
-    colorama_stdout = colorama.AnsiToWin32(sys.stdout).stream
-
-elif sys.platform == "win32":
-    colorama_wrap = False
-
-colorama.init(wrap=colorama_wrap)
+logging_stdout = sys.stdout
+if sys.platform == "win32":
+    if not os.getenv("TERM"):
+        logging_stdout = colorama.AnsiToWin32(sys.stdout).stream
+    colorama.init(wrap=False)
```

Comment:
And I think this logic is the minimal reduction of the logic we had before. Again, please correct any logical errors in this refactoring